### DISCUSSION TITLE
Whisper terminal all output_format files are different than >  command redirection, how to only use redirection?

### DISCUSSION THREAD

**Discussion by dimyself** [openai/whisper#1649] [Q&A] [answered]
Hi there! 

Ok , so I tried all output_formats, and none of them are the same as just using good old command redirection > into a txt file

The difference is redirecting the command output has the text on the same lines as the timestamps. 

Here's an example of the output I want (the command output in the terminal):

> [03:19:13.080 --> 03:19:16.120]  text 1
> [03:19:21.320 --> 03:19:28.440]  text 2
> [03:19:28.440 --> 03:19:32.600]  text 3
> etc...
 
For my case, the above ^ is easier to deal with and search with the text on the same line as the timestamps (rather than text on the next line after the timestamps). I can just use something simPle like grep

Is there a way to get this kind of output using output_format? I checked all the options: {txt,vtt,srt,tsv,json,all} ... and none of them were the same as the command output. 

If possilbe, i would like output_format to have a text option with the same output as the command output

OR

A way to disable output_format and only use command redirection would also work

Otherwise, I have to do unnecessary steps (create an output_format I wont use and then delete it after running each time)

Thank you!

**Comment by glangford** [comment, marked as answer]
The command output you are seeing in the terminal comes from the `--verbose` option. It is mostly a log intended to show progress, and isn't really in the same category as the other output formats.

So, since there isn't an output format of `none` at the moment you need to create an output format you won't use.

**Reply by glangford** [reply]
...the other way to address this (for now) would be to use Python rather than the command line.

**Comment by dimyself** [comment]
ok thanks i appreciate it... maybe in the future they can implement this

**Comment by ryanheise** [comment]
> For my case, the above ^ is easier to deal with and search with the text on the same line as the timestamps (rather than text on the next line after the timestamps). I can just use something simPle like grep

I don't think Whisper will support non-standard output formats. But if you're already using the command line and things like grep, then it should be easy to use the command line to convert an SRT file into the format you want with:

```bash
# This assumes a language where lines can be joined with a space delimiter.
cat file.srt | awk 'BEGIN { RS = "" } { gsub(/\n/, " "); print }'
```

You could pipe that into grep as you are currently doing.
